modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/UpdateLogImpl.java

```diff
@@ -225,6 +225,8 @@ public CompletableFuture<Boolean> saveSnapshot(SnapshotEntry update) {
 
             Iif iif = iif(versionIsRecent, saveSnapshotAndDropOutdatedUpdates, ops().yield(false));
 
+            LOG.info("Snapshot version is updated. [Old version: {}, new version: {}]", oldSnapshotVersion, snapshotVersion);
```

Comment:
```suggestion
            LOG.info("Snapshot version is updated [oldVersion={}, newVersion={}].", oldSnapshotVersion, snapshotVersion);
```

Reply:
btw, this message is not quite correct. Next invoke is conditional, so there is chance that update won't be applied.

Reply:
Removed this method.